frontend/src/components/Toggle/Toggle.tsx

```diff
@@ -58,9 +79,28 @@ export const Toggle = forwardRef<ToggleProps, 'input'>(
       <Flex __css={{ ...styles.overallContainer, ...containerStyles }}>
         {(label || description) && (
           <Box __css={styles.textContainer}>
-            <FormLabel.Label sx={{ ...styles.label, ...labelStyles }}>
-              {label}
-            </FormLabel.Label>
+            <Flex alignItems="center">
```

Comment:
no / minimal changes to other usages of `Tooltip` expected 

Rationale: 
- flex direction is row, with no changes to the main axis orientation 
- tooltipText is checked before adding another component